clients/drcachesim/tests/prefetch_analyzer_launcher.cpp

```diff
@@ -0,0 +1,84 @@
+/* **********************************************************
+ * Copyright (c) 2020 Google, Inc.  All rights reserved.
+ * **********************************************************/
+
+/*
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * * Neither the name of Google, Inc. nor the names of its contributors may be
+ *   used to endorse or promote products derived from this software without
+ *   specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL VMWARE, INC. OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+ * DAMAGE.
+ */
+
+/* Standalone prefetch analyzer tool launcher for file traces. */
```

Comment:
Hmm, I think I suggested originally putting the analyzer into a static start/stop test, which would have the whole test all in one place and not require multiple binaries to be built, with no separate special launcher or separate analyzer.  Did you end up deciding against that?  What was the reasoning?  We currently have tests that run raw2trace after gathering: is there a blocker to running an analyzer there?  I would think the analyzer would have fewer issues than raw2trace which has some link complexities on Windows.

Reply:
Based on the latest update on #1551, I think start/stop support is not implemented yet for ARM. As discussed offline, existing burst_* tests that use the start/stop api are also not enabled for AArch64 yet due to #2007.

third_party/dynamorio/trunk/suite/tests/CMakeLists.txt:3408
`# FIXME i#2007: fails to link on A64`

Reply:
Hmm, ok, the #1551 is 32-bit arm.  I recall running attach/detach on aarch64 and it working -- but yes on the Travis setup there was some link issue (fixed on machines we control w/ a binutils tweak).  Sure, since you already have this let's go with this.